data/presets/highway/track.json

```diff
@@ -30,29 +30,34 @@
     "terms": [
         "4wd",
         "4x4",
-        "agricultural road",
         "atv",
-        "carriage road",
         "double track",
-        "farm road",
-        "fire road",
-        "forest road",
         "four wheel drive",
         "high-clearance",
         "jeep",
-        "land access road",
-        "logging road",
         "offroad",
+        "off-road",
         "overgrown",
         "primitive",
         "quad",
-        "ranch road",
         "rut",
-        "track",
-        "track road",
-        "two track",
-        "unmaintained",
-        "woods road"
+        "unmaintained"
     ],
-    "name": "Track / Land-Access Road"
+    "name": "Track / Land-Access Road",
+    "aliases": [
+        "Agricultural Road",
+        "Carriage Road",
+        "Dirt Road",
```

Comment:
I don't think "Dirt Road" should be included as an alias.  In American English this term is used to mean any unpaved road regardless of purpose or importance.  Many of these roads are properly tagged with a different `highway` class and `surface=dirt/compacted/unpaved`.

Reply:
I'd say the likelyhood that an unmaintained road would be `track` is much higher than the likelyhood that a dirt road would be track.  I live in a state where more of the public road network is dirt than paved ([Vermont road stats](https://vtrans.vermont.gov/planning/maps/stats)).  These are roads people live on that are part of the general purpose road network, so here it is just as likely that a given dirt road should be tagged `highway=residential/unclassified` + `surface=unpaved` and `highway=track` should not be assumed just because a road is dirt.

I'm not really sure what the difference is between "aliases" and "terms", but I'd suggest not including the term "dirt road" in either to avoid confusion.